fjord/feedback/tests/test_views.py

```diff
@@ -115,68 +117,60 @@ def test_valid_sad(self):
         eq_(u'14.0.1', feedback.version)
 
     @with_waffle('thankyou', True)
-    def test_use_thank_you_sad(self):
-        """Thanks_sad template is used when appropriate"""
+    def test_thankyou_flag_active(self):
+        """Verify response and suggestions when thankyou flag is active"""
         url = reverse('feedback', args=(u'firefox',), locale='en-US')
         r = self.client.post(url, {
             'happy': 0,
-            'description': u"I want to know why Firefox doesn't make me sandwiches!",
+            'description': u'Why Firefox not make me sandwiches!',
```

Comment:
Oh. I thought it was just to make the sentence more like normal Input feedback. Now that I know there is a technical reason for it, I'm not as excited.